src/main/java/io/confluent/connect/elasticsearch/Validator.java

```diff
@@ -117,6 +121,16 @@ private void validateCredentials() {
     }
   }
 
+  private void validateDataStreamFields() {
```

Comment:
lets also add a ticket to make a validator for ES 7.9

Reply:
Added: https://confluentinc.atlassian.net/browse/CCMSG-1028